circuits/lib/utils/claimUtils.circom

```diff
@@ -147,6 +147,36 @@ template verifyCredentialSubject() {
     subjectOtherIden.id === id;
 }
 
+// verifyCredentialSubject verifies that claim is issued to a specified identity or identity profile
+// if nonce 0 is used, the claim should be issued to the genesis identity
+template verifyCredentialSubjectProfile() {
+	signal input claim[8];
+	signal input id;
+	signal input nonce;
+
+	component header = getClaimHeader();
+	for (var i=0; i<8; i++) { header.claim[i] <== claim[i]; }
+
+	component subjectOtherIden = getClaimSubjectOtherIden();
+	for (var i=0; i<8; i++) { subjectOtherIden.claim[i] <== claim[i]; }
+
+
+	/* ProfileID calculation */
```

Comment:
Something wrong with the tabulation here

Reply:
fixed